server/models/User.js

```diff
@@ -0,0 +1,27 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const UserSchema = new Schema({
+	firstName: {
+		type: String,
+		required: true,
+		minlength: 2
+	},
+	lastName: {
+		type: String,
+		required: true,
+		minlength: 2
+	},
+	email: {
+		type: String,
+		required: true,
+		minlength: 5,
+		unique: true
+	},
+	pass: {
+		type: String,
+		required: true
+	}
+}, {collection: 'users'});
+
+module.exports = User = mongoose.model('users', UserSchema);
```

Comment:
It would be cleaner to break this into two lines. Additionally better practice would be to name the collection "User" to match conventions.